.gitignore

```diff
@@ -1 +1,3 @@
 *.sw?
+**/parallel-extensions/*.crx
```

Comment:
What's the point of the leading "**"?

Reply:
It's a Windows thing iirc, though since badger sett doesn't yet work in Windows, I took it out in the recent update.

Reply:
Looking up "gitignore syntax" leads to https://git-scm.com/docs/gitignore, which says the following:

>A leading "`**`" followed by a slash means match in all directories. For example, "`**/foo`" matches file or directory "`foo`" anywhere, the same as pattern "`foo`". "`**/foo/bar`" matches file or directory "`bar`" anywhere that is directly under directory "`foo`".

Elsewhere on that page:

>a leading slash is not relevant if there is already a middle slash in the pattern

Reply:
Good to know! The current version of the gitignore for this PR matches for all `.crx` and all `.zip` files in the `parallel-extensions/` directory. There's no leading slash or `**`